dapps/assets/src/components/Assets.ts

```diff
@@ -30,6 +30,19 @@ declare module 'vue/types/vue' {
     $router: VueRouter;
     $route: Route;
     $store: any;
+    $t: Function;
+    dapp: Dapp;
+    getRuntime: Function;
+  }
+
+  interface Dapp {
```

Comment:
should we add this to an interfaces file and just import it here? I guess it's commonly used.

Reply:
Normally this would be done in the component.ts and be available to all EvanComponents. Problem is currently modules arent exported properly so compiler doesn't understand. This should be fixed with TS-fixing instead.